src/components/common/icons/tickets/index.tsx

```diff
@@ -0,0 +1,11 @@
+const TicketsIcon = () => (
```

Comment:
Why it is here? It should be in the assets folder

Reply:
Deleted. (Icons folder in common is also deleted since I import svgs as ReactComponents from assets directly, so these components here are never used)